jwst/background/subtract_images_step.py

```diff
@@ -1,8 +1,8 @@
 #! /usr/bin/env python
 
-from jwst.stpipe import Step
+from ..stpipe import Step
 from .. import datamodels
-from jwst.background import subtract_images
+from .. import subtract_images
```

Comment:
I think this one might actually need to be at the same relative level, i.e. "from . import subtract_images". I believe they're in the same directory.